### PR TITLE
Publish November RubyGems Monthly Updates

### PR DESCRIPTION
**Pull request by lauragift21**
RubyGems monthly update roundup for November 2023

### CODE REVIEW

File: _posts/2023-12-12-november-rubygems-updates.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- extracted verified session logic into a concern - [#4210](https://github.com/rubygems/rubygems.org/pull/4210).
+- updated installation instructions OS X - [#4203](https://github.com/rubygems/rubygems.org/pull/4257).
+- upgraded shoryuken to version 5.x. - [#4166](https://github.com/rubygems/rubygems.org/pull/4166).
+
+
+![Trusted Publishers Demo](https://private-user-images.githubusercontent.com/1946610/289184337-b025c966-f4c2-45df-926a-1f72048e8fc3.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI0NTU2NjEsIm5iZiI6MTcwMjQ1NTM2MSwicGF0aCI6Ii8xOTQ2NjEwLzI4OTE4NDMzNy1iMDI1Yzk2Ni1mNGMyLTQ1ZGYtOTI2YS0xZjcyMDQ4ZThmYzMuZ2lmP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQUlXTkpZQVg0Q1NWRUg1M0ElMkYyMDIzMTIxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyMzEyMTNUMDgxNjAxWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9N2MxNDQ0ZWI2NzUzMWFhYTM1YmRlOTlhZDNmYWM3MDEyMGRhM2I4NTZkYjdhZjMxZDIyMzI5ODIyZDc0ZjRiYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.mW0_pyeU3njaWbcWNFQAnyFim5nlZirsJT9MqerSUKk)
```
Comment (segiddins) [inline _posts/2023-12-12-november-rubygems-updates.md:56]: instead let's link to the blog post? https://github.com/rubygems/rubygems.github.io/pull/173

File: _posts/2023-12-12-november-rubygems-updates.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+
+In November, RubyGems gained [110 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-11-01%7D...master@%7B2023-11-31%7D) contributed by 22 authors. There were 1,868 additions and 56,823 deletions across 1,485 files.
+
+## RubyGems.org News
+
+The updates to RubyGems.org this month reflect a strong commitment to improving user experience, enhancing security, and modernizing the platform. Here’s a brief overview of the key improvements in the release:
+
+- fixed deletion `version_id` backfill when a user is deleted - [#4259](https://github.com/rubygems/rubygems.org/pull/4259).
+- upgrade to Postgresql 12 and updated production and staging `DATABASE_URL` to PostgreSQL 12 instance - [#4245](https://github.com/rubygems/rubygems.org/pull/4257), [#4256](https://github.com/rubygems/rubygems.org/pull/4256).
```
[the remark this answers is not in the document]
Comment (martinemde) [reply _posts/2023-12-12-november-rubygems-updates.md:44]: @simi that makes sense to me. Could you write up a paragraph that tells a short explanation of the upgrade process? I think people will be interested to hear about that.

**Review by martinemde — APPROVED** [review]
I think @simi and @segiddins already called out the two things worth updating. I don't have anything else to add. Thank you!

File: _posts/2023-12-12-november-rubygems-updates.md (hunk 1 above)
Comment (segiddins) [reply _posts/2023-12-12-november-rubygems-updates.md:56]: https://blog.rubygems.org/2023/12/14/trusted-publishing.html

Comment (lauragift21) [reply _posts/2023-12-12-november-rubygems-updates.md:56]: Thanks @segiddins! I will link to the blog post.

File: _posts/2023-12-12-november-rubygems-updates.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lauragift21) [reply _posts/2023-12-12-november-rubygems-updates.md:44]: Yes, I can move this out of this list to its own paragraph. @simi, can you help share a paragraph to explain the upgrade process?

**Review by martinemde — APPROVED** [review]

**Pull request merged**